handout/a11y/key-concerns/keyboard-accessibility.md

```diff
@@ -0,0 +1,55 @@
+# Keyboard Accessibility
+
+Keyboard accessibility is the ability of your application to be interacted with using just a keyboard. The more streamlined the site can be used this way, the more keyboard accessible it is. Keyboard accessibility is one of the largest aspects of web accessibility since it targets both those with motor disabilities who can't use a mouse, blind users who typically prefer to navigate websites with their keyboard and those who prefer not to use a mouse.
```

Comment:
I like the first sentence as an intro to the keyboard section but agreed on using a bullet list